src/njs_builtin.c

```diff
@@ -920,6 +920,8 @@ njs_top_level_constructor(njs_vm_t *vm, njs_object_prop_t *self,
         ctor = &njs_vm_ctor(vm, njs_prop_magic16(self));
 
         njs_set_function(retval, ctor);
+
+        return NJS_OK;
```

Comment:
Looks like it introduce performance decrease for all read accesses to all top_level_constructors